src/hexo/source/_data/contributors.yml

```diff
@@ -1,45 +1,60 @@
 - name: Antón Molleda
   company: Microsoft
   avatar: https://avatars.githubusercontent.com/molant
+  website: https://github.com/molant
```

Comment:
Why not just use `github`, and have `website` as a separate thing? 

Cc @sonarwhal/contributors